### PR TITLE
Blog: week 48 (November 29th - December 3rd)

### PR DESCRIPTION
**Pull request by FrNecas**

### CODE REVIEW

**Review by TomasTomecek — APPROVED** [review]

File: content/posts/November-2021.md [hunk 1]
```diff
@@ -48,3 +48,23 @@ weight: 69
   needs to be present in the dist-git for now
   (the state for the new commit is used).
   ([packit-service#1278](https://github.com/packit/packit-service/pull/1278))
+
+## Week 48 (November 29th - December 3rd)
+
+- `oraclelinux-8` distro specification is now mapped to Oracle-Linux-8.5
+  compose.
+  ([packit-service#1301](https://github.com/packit/packit-service/pull/1301))
```
Comment (jpopelka) [inline content/posts/November-2021.md:56]: I'd omit this, it's just for the internal testing farm (which only 2 projects use) and it's a tiny detail.

File: content/posts/November-2021.md [hunk 2]
```diff
@@ -48,3 +48,23 @@ weight: 69
...
+- External contributors can't trigger internal tests initially. Project
+  maintainers need to trigger the action via `/packit test` comment to run
+  the job.
+  ([packit-service#1302](https://github.com/packit/packit-service/pull/1302))
+  ([packit-service#1305](https://github.com/packit/packit-service/pull/1305))
```
[the remark this answers is not in the document]
Comment (FrNecas) [reply content/posts/November-2021.md:61]: Hm, depends on how you look at it. From looking into 1305, the only current use of the new feature is related to this change (it informs in MD about the need to run `/packit test`). However, the change is definitely more general but I feel like that's more of an internal thing not relevant to the users.

File: content/posts/November-2021.md (hunk 1 above)
Comment (FrNecas) [reply content/posts/November-2021.md]: Good point, I was a bit torn whether to include it or not but since it's so tiny, I've dropped it

Comment (TomasTomecek) [reply content/posts/November-2021.md]: I find it funny that our release notes mention oraclelinux, so I'd keep it for sake of amusement :)

**Comment by FrNecas** [comment]
Eh, forgot a bit about this. Merging...

**Pull request merged**